backend/api/Models/Concepts/Property/GeometryMap.cs

```diff
@@ -12,7 +12,7 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.Coordinate, src => src.Coordinate);
 
             config.NewConfig<GeometryModel, Geometry>()
-                .ConstructUsing(src => GeometryHelper.CreatePoint(src.Coordinate.X, src.Coordinate.Y));
+                .ConstructUsing(src => GeometryHelper.CreatePoint(src.Coordinate.X, src.Coordinate.Y, 4326));
```

Comment:
nit: would be better to use a constants instead of a number

Reply:
good idea - done